crypto/nrf_oberon/lib/license.txt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
I haven't seen these types of updates before (maybe its just me), shouldn't this either be the original value or `2018-2020` ? I have no idea what I'm talking about here, but I guess its worth double checking.

Reply:
Double checked. 2020 is OK